bigquery/config/template/dataset.json

```diff
@@ -1,7 +1,11 @@
 {
-  // Elasticsearch index will be given this name.
+  // Dataset name. For Project Baseline (https://www.projectbaseline.com/), this
+  // could be "Baseline" or "Project Baseline". This name will be displayed in
+  // the Data Explorer UI. Also, the Elasticsearch index will be given this
+  // name.
```

Comment:
Is this right? I ran into problems with "AMP PD":

`elasticsearch.exceptions.RequestError: TransportError(400, u'invalid_index_name_exception', u'Invalid index name [AMP PD], must not contain the following characters [ , ", *, \\, <, |, ,, >, /, ?]')
`

and then:

`elasticsearch.exceptions.RequestError: TransportError(400, u'invalid_index_name_exception', u'Invalid index name [AMP-PD], must be lowercase')
`

Could you add details about the character restrictions?

Reply:
Done. Also filed #5.